packages/iota/test/validation/outputs.spec.ts

```diff
@@ -55,4 +58,112 @@ describe("Features", () => {
             "Basic output amount field must be larger than zero."
         ]));
     });
+
+    test("Can validate alias output", () => {
+        const output: IAliasOutput = {
+            type: ALIAS_OUTPUT_TYPE,
+            amount: "455655655",
+            aliasId: "0xb6b82443901a2ab6beefdcb88acff1ca359f211a474cb50cf63aa6a24721f9aa",
+            stateIndex: 1,
+            foundryCounter: 0,
+            unlockConditions: [
+                {
+                    type: STATE_CONTROLLER_ADDRESS_UNLOCK_CONDITION_TYPE,
+                    address: {
+                        type: 8,
+                        aliasId: "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"
+                    }
+                },
+                {
+                    type: GOVERNOR_ADDRESS_UNLOCK_CONDITION_TYPE,
+                    address: {
+                        type: 8,
+                        aliasId: "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"
+                    }
+                }
+            ]
+        };
+
+        const result = validateOutputs([output], protocolInfo);
+        expect(result.isValid).toEqual(true);
+    });
+
+    test("Fail validate alias output", () => {
+        const output: IAliasOutput = {
+            type: ALIAS_OUTPUT_TYPE,
+            amount: "0",
+            aliasId: "0xb6b82443901a2ab6beefdcb88acff1ca359f211a474cb50cf63aa6a24721f9aa",
+            stateIndex: 0,
+            foundryCounter: 0,
+            unlockConditions: []
+        };
+
+        const result = validateOutputs([output], protocolInfo);
+        expect(result.isValid).toEqual(false);
+        expect(result.errors).toEqual(expect.arrayContaining([
+            "Alias output amount field must be larger than zero.",
+            "Unlock conditions count must be equal to 2.",
+            "State controller address unlock condition or Governor address unlock condition must be present."
+        ]));
+    });
+
+    test("Fail same unlock condition address as alias id", () => {
```

Comment:
Change the test name to: "should fail on unlock condition address same as alias id".